api/app/api.py

```diff
@@ -1,22 +1,28 @@
 from fastapi import FastAPI, Request
+from decouple import config
 from fastapi.middleware.cors import CORSMiddleware
 from fastapi.encoders import jsonable_encoder
 from fastapi_mail import FastMail, MessageSchema,ConnectionConfig
 import json
 from app import EMAIL, PSWRD
 import os
 
+
+
 app = FastAPI()
 
+EMAIL = os.getenv("EMAIL")
+PSWRD = os.getenv("PSWRD")
+
 origins = [
     "http://localhost:3000",
     "localhost:3000",
     "*"
 ]
 
 conf = ConnectionConfig(
-    MAIL_USERNAME = EMAIL,
-    MAIL_PASSWORD = PSWRD,
+    CONTACT_EMAIL = EMAIL,

```

Comment:
Did you run the changes and check? CONTACT_EMAIL is not a parameter for the Connection Config function